service/configure.ac

```
@@ -327,6 +339,8 @@ AX_CHECK_COMPILE_FLAG([-fp-model="precise"], [NO_FASTMATH="-fp-model=precise"],
 
 [EXTRA_CFLAGS="$EXTRA_CFLAGS -Wall -Werror $NO_FASTMATH"]
 [EXTRA_CXXFLAGS="$EXTRA_CXXFLAGS -Wall -Werror $NO_FASTMATH"]
+[LIBS="$LIBS $GRPC_LIBS"]
+[INCLUDE="$INCLUCE $GRPC_INCLUDE"]
```

```suggestion
[INCLUDE="$INCLUDE $GRPC_INCLUDE"]
```

Is INCLUDE an automake thing or a make thing, or a geopm builder thing? I can't find it in the user manuals. I assume one layer uses that to add `-I<paths>` to relevant compiler flag variables.